numpy/ma/extras.py

```diff
@@ -845,6 +845,14 @@ def compress_nd(x, axis=None):
     -------
     compress_array : ndarray
         The compressed array.
+
+    Examples
+    --------
+    >>> x = np.ma.array(np.arange(6).reshape(3, 2), mask=[[1, 0], [0, 0], [0, 0]])
+
+    >>> np.ma.compress_nd(x)
+    array([[7, 8]])
```

Comment:
There's something wrong here: the input array doesn't contain the values 7, 8.

Reply:
Sorry, The x should be x = np.ma.array(np.arange(9).reshape(3, 3), mask=[[1, 0, 0],[1,0,0],[0,0,0]]. I will fix it now.